### ISSUE TITLE
classifier training hangs for 2400 images

### ISSUE THREAD

**Issue by jaiatncson7** [davidsandberg/facenet#1116]
I am testing the classifier with 2400 images. it just hang in model.fit(emb_array, labels) function call indefinitely.

The same code works fine for 1920 images in raw_faces. Any idea how to resolve this issue? Please help.

**Comment by chenyyxx** [comment]
I also faced the same issue. Any idea how to resolve this? Please help!
